### PR TITLE
style: bg update

### PR DESCRIPTION
**Pull request by ruturaj-gh**

### CODE REVIEW

**Comment by ruturaj-gh** [comment]
Updated the UI
Hapy Hactoberfest

File: assets/js/script.js [hunk 1]
```diff
@@ -143,7 +143,10 @@ function checkResult() {
 
 function isWinner(a,b){
     if (a == 3 || b == 3) {
+        
         document.getElementById("result").innerHTML = a == 3 ? "Player 1 Win" : "Player 2 Win";
+        if(document.getElementById("result").innerHTML=="Player 1 Win")  alert("Player 1 Win Hurray !")
+        else alert("Player 2 win Hurray !");
```
Comment (malivinayak) [inline assets/js/script.js:149]: As soon as any player wins it first give alert message and then update the box for X or O. after that innerhtml of result update as follows `document.getElementById("result").innerHTML = a == 3 ? "Player 1 Win" : "Player 2 Win";`

As soon as click box located at 3rd row 1st column it gives winning alert
image

After Clicking OK, it update the X or O sign in box
image

**Review by malivinayak — CHANGES_REQUESTED** [review]

File: assets/css/index.css [hunk 2]
```diff
@@ -24,4 +29,15 @@
...
+  
+  .box:hover {
+    background-color: rgba(var(--aliceblue), 0.5);
+  }
+  
+  .box:active {
+    background-color: var(--aliceblue);
+    border-radius: 5px;
+    margin: 0;
+  }
```
Comment (malivinayak) [inline assets/css/index.css:43]: Add extra new line at end

File: assets/css/index.css [hunk 3]
```diff
@@ -1,20 +1,25 @@
...
     flex-direction: column;
     justify-content: center;
     align-items: center;
     height: 90vh;
-}
-
-.playground {
+  }
+  
+  .playground {
```
Comment (malivinayak) [inline assets/css/index.css:14]: remove extra tab space added

File: assets/css/index.css [hunk 4]
```diff
@@ -1,20 +1,25 @@
...
-    background-color: aliceblue;
+    background-color: var(--aliceblue);
     padding: 10px;
     width: 33rem;
-}
-
-.box {
+  }
+  
+  .box {
```
Comment (malivinayak) [inline assets/css/index.css:22]: remove extra tab space added

File: assets/js/script.js [hunk 5]
```diff
@@ -143,7 +143,10 @@ function checkResult() {
 
 function isWinner(a,b){
     if (a == 3 || b == 3) {
+
```
Comment (malivinayak) [inline assets/js/script.js:146]: Remove extra new line

**Review by malivinayak — CHANGES_REQUESTED** [review]
As a part of code beautification make the suggested changes so that I can merge the PR

File: assets/css/index.css (hunk 2 above)
Comment (ruturaj-gh) [reply assets/css/index.css:43]: Ok i'll fix it

Comment (ruturaj-gh) [reply assets/css/index.css:43]: @malivinayak should i remove alert box or render innerhtml first and then alert statement after 1 second delay

File: assets/js/script.js (hunk 1 above)
Comment (malivinayak) [reply assets/js/script.js]: This issue is not resolved yet
Checkout the video attached to latest comment

File: assets/css/index.css (hunk 2 above)
Comment (malivinayak) [reply assets/css/index.css]: Yes you can take necessary steps to resolve issue

**Comment by malivinayak** [comment]
Don't think like I am giving too much changes.
Just meet the requirements properly I will merge PR.
You are working as open source so you must follow the guidelines.

File: assets/css/index.css
Comment (malivinayak) [inline assets/css/index.css:1]: Add new line between each css property

e.g.

<table>
  <tr>
  </tr>
  <tr>
    <td align="center"> &nbsp;  &nbsp;  &nbsp;  &nbsp;  &nbsp;  &nbsp;  &nbsp;  &nbsp;  &nbsp;  &nbsp;  &nbsp;  &nbsp;  &nbsp;  &nbsp;  &nbsp;  &nbsp;  &nbsp; <b>Current : </b> &nbsp;  &nbsp;  &nbsp;  &nbsp;  &nbsp;  &nbsp;  &nbsp;  &nbsp;  &nbsp;  &nbsp;  &nbsp;  &nbsp;  &nbsp;  &nbsp;  &nbsp;  &nbsp;  &nbsp;  </td> 
    <td align="center"> &nbsp;  &nbsp;  &nbsp;  &nbsp;  &nbsp;  &nbsp;  &nbsp;  &nbsp;  &nbsp;  &nbsp;  &nbsp;  &nbsp;  &nbsp;  &nbsp;  &nbsp;  &nbsp;  &nbsp;  <b>Expected : </b>&nbsp;  &nbsp;  &nbsp;  &nbsp;  &nbsp;  &nbsp;  &nbsp;  &nbsp;  &nbsp;  &nbsp;  &nbsp;  &nbsp;  &nbsp;  &nbsp;  &nbsp;  &nbsp;  &nbsp;  </td>
  </tr>

 <tr>
<td> 

```css
    :root {
      --aliceblue: aliceblue; . . .
    }
    .parent {
      display: flex; . . .
    }
    .playground {
      display: grid; . . .
    }
```

</td>

<td>
 
```css
:root {
  --aliceblue: aliceblue; . . .
}

.parent {
  display: flex; . . .
}

.playground {
  display: grid; . . .
}
```

 </td>
 </tr>

</table>

<br>

Changes expecting:
 
```css
:root {
  --aliceblue: aliceblue; . . .
}
/* Blank new line*/
.parent {
  display: flex; . . .
}
/* Blank new line*/
.playground {
  display: grid; . . .
}
```

**Review by malivinayak — CHANGES_REQUESTED** [review]

File: assets/js/script.js [hunk 6]
```diff
@@ -130,26 +128,39 @@ function checkResult() {
         }
     }
     isWinner(d1, d2);
-
     // Check for a draw
     if (emptyCells === 0) {
         document.getElementById("result").innerHTML = "It's a Draw!";
-        setTimeout(function () {
-            window.location.reload(); // Reload the window after a brief delay
-        }, 2000); // You can adjust the delay time (in milliseconds) as needed
```
Comment (malivinayak) [inline assets/js/script.js:139]: After draw window must be reload after a brief delay just like after winning condition

**Review by malivinayak — CHANGES_REQUESTED** [review]

File: assets/js/script.js (hunk 6 above)
Comment (ruturaj-gh) [reply assets/js/script.js:139]: Can you explicitly define the delay time?

Comment (malivinayak) [reply assets/js/script.js:139]: keep either 2000 or 3000
You have deleted the above code which causes the issue after draw

**Review by malivinayak — APPROVED** [review]
task don : bg update

**Comment by ruturaj-gh** [comment]
image
??

**Comment by ruturaj-gh** [comment]
Maintainers need to add the `hacktoberfest` topic to the repository, or add the `hacktoberfest-accepted` label to your PR/MR.
i think you need to add hacktoberfest label too.

**Comment by malivinayak** [comment]
Refresh the page for 5-15 min and then let me know the status

**Comment by ruturaj-gh** [comment]
thank you !!

**Pull request merged**